### PR TITLE
Added descriptions and examples for font-style.

### PR DESCRIPTION
**Pull request by SarahCarlock**
### Description

Added a description and method example for font-styles' normal, italic, oblique, inherit, and initial. Provided two images showing font-style used in an HTML document and it's output. 

### Issue Solved

- Closes issue #4416 

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [ x] My entry follows the Codecademy Docs style guide.
- [ x] My changes generate no new warnings.
- [ x] I have performed a self-review of my own writing and code.
- [ x] I have checked my entry and corrected any misspellings.
- [ x] I have made corresponding changes to the documentation if needed.
- [ x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/css/concepts/typography/terms/font-style/font-style.md [hunk 1]
```diff
@@ -7,13 +7,18 @@ Subjects:
...
   - 'paths/front-end-engineer-career-path'
   - 'paths/full-stack-engineer-career-path'
 ---
 
-To set the font style in which text will appear.
+Changing the **font-style** selection will determine if the font is vertical or has a degree of slant. The most commonly used varient of font-style is `italic`.
+
+![font-styles in html.](https://github.com/SarahCarlock/codecademyDocs/assets/112113100/4dce07d9-bd71-447c-94c6-d8468d66e150)
+
+![font-styles visual examples.](https://github.com/SarahCarlock/codecademyDocs/assets/112113100/fcb3817f-86ac-47f5-a44e-386da8ad439a)
```
Comment (ishg-153) [inline content/css/concepts/typography/terms/font-style/font-style.md:21]: For this do the following things
- Move this to the end of the entry after all the examples.
- Change the link to this format: `https://raw.githubusercontent.com/Codecademy/docs/main/media/` followed by the image name. For this, you'll have to add your image to the media folder on this branch.

File: content/css/concepts/typography/terms/font-style/font-style.md [hunk 2]
```diff
@@ -24,17 +29,49 @@ font-style: <font-style-value>;
...
 
 - `normal`
-- `oblique`
 - `italic`
+- `oblique`
 - `inherit`
 - `initial`
 
-## Example 1
+## Examples
```
Comment (ishg-153) [inline content/css/concepts/typography/terms/font-style/font-style.md:37]:
```suggestion
## Example
```

File: content/css/concepts/typography/terms/font-style/font-style.md [hunk 3]
```diff
@@ -24,17 +29,49 @@ font-style: <font-style-value>;
...
+
+### `normal`: text will be in it's basic design. This is the default style in Cascading Style Sheets (CSS).
```
Comment (ishg-153) [inline content/css/concepts/typography/terms/font-style/font-style.md:39]:
```suggestion
`normal`: text will be in it's basic design. This is the default style in Cascading Style Sheets (CSS).
```

File: content/css/concepts/typography/terms/font-style/font-style.md [hunk 4]
```diff
@@ -24,17 +29,49 @@ font-style: <font-style-value>;
...
+
+```css
+p {
+  font-style: normal;
+}
+```
 
-Set the `p` tag to `italic`:
+### `italic`: _text will be tilted, some characters will be replaced with a specialized tilted character (e.g. "a" to "ɑ" in the font Calibri)._
```
Comment (ishg-153) [inline content/css/concepts/typography/terms/font-style/font-style.md:47]:
```suggestion
`italic`: text will be tilted, some characters will be replaced with a specialized tilted character (e.g. "a" to "ɑ" in the font Calibri).
```

File: content/css/concepts/typography/terms/font-style/font-style.md [hunk 5]
```diff
@@ -24,17 +29,49 @@ font-style: <font-style-value>;
...
 
 ```css
 p {
   font-style: italic;
 }
 ```
+
+### `oblique`: gives text a minor slant to the font without changing any characters. It is less supported and may not always be available.
```
Comment (ishg-153) [inline content/css/concepts/typography/terms/font-style/font-style.md:55]:
```suggestion
`oblique`: gives text a minor slant to the font without changing any characters. It is less supported and may not always be available.
```

File: content/css/concepts/typography/terms/font-style/font-style.md [hunk 6]
```diff
@@ -24,17 +29,49 @@ font-style: <font-style-value>;
...
+
+```css
+p {
+  font-style: oblique;
+}
+```
+
+### `inherit`: text will take on the style used by the parent element.
```
Comment (ishg-153) [inline content/css/concepts/typography/terms/font-style/font-style.md:63]:
```suggestion
`inherit`: text will take on the style used by the parent element.
```

File: content/css/concepts/typography/terms/font-style/font-style.md [hunk 7]
```diff
@@ -24,17 +29,49 @@ font-style: <font-style-value>;
...
+
+```css
+p {
+  font-style: inherit;
+}
+```
+
+### `initial`: sets the text font-style to it's default style, similar to `normal`.
```
Comment (ishg-153) [inline content/css/concepts/typography/terms/font-style/font-style.md:71]:
```suggestion
`initial`: sets the text font-style to it's default style, similar to `normal`.
```

**Review by ishg-153 — CHANGES_REQUESTED** [review]
@SarahCarlock I have suggested changes. Please make them and let me know once done.

File: content/css/concepts/typography/terms/font-style/font-style.md [hunk 8]
```diff
@@ -7,13 +7,18 @@ Subjects:
...
 CatalogContent:
   - 'learn-css'
   - 'paths/front-end-engineer-career-path'
   - 'paths/full-stack-engineer-career-path'
 ---
 
-To set the font style in which text will appear.
+Changing the **font-style** selection will determine if the font is vertical or has a degree of slant. The most commonly used varient of font-style is `italic`.
+
+![font-styles in html.](https://github.com/SarahCarlock/codecademyDocs/assets/112113100/4dce07d9-bd71-447c-94c6-d8468d66e150)
```
[the remark this answers is not in the document]
Comment (SarahCarlock) [reply content/css/concepts/typography/terms/font-style/font-style.md:19]: Removed

**Comment by SarahCarlock** [comment]
@ishg-153 I've completed the changes requested.

File: content/css/concepts/typography/terms/font-style/font-style.md (hunk 1 above)
Comment (ishg-153) [reply content/css/concepts/typography/terms/font-style/font-style.md]: Yep!

**Review by ishg-153 — APPROVED** [review]
@SarahCarlock I made some changes from my side. This is ready for second review!

File: content/css/concepts/typography/terms/font-style/font-style.md [hunk 9]
```diff
@@ -24,17 +25,51 @@ font-style: <font-style-value>;
...
-- `oblique`
 - `italic`
+- `oblique`
 - `inherit`
 - `initial`
 
-## Example 1
+## Example
+
+`normal`: text will be in it's basic design. This is the default style in Cascading Style Sheets (CSS).
```
Comment (avdhoottt) [inline content/css/concepts/typography/terms/font-style/font-style.md:35]:
```suggestion
`normal`: The text will be in its basic design. This is the default style in Cascading Style Sheets (CSS).
```

File: content/css/concepts/typography/terms/font-style/font-style.md [hunk 10]
```diff
@@ -24,17 +25,51 @@ font-style: <font-style-value>;
...
+
+```css
+p {
+  font-style: normal;
+}
+```
 
-Set the `p` tag to `italic`:
+`italic`: text will be tilted, and some characters will be replaced with a specialized tilted character (e.g. "a" to "ɑ" in the font Calibri).
```
Comment (avdhoottt) [inline content/css/concepts/typography/terms/font-style/font-style.md:43]:
```suggestion
`italic`: text will be tilted, and some characters will be replaced with a specialized tilted character (e.g. "a" to "ɑ" in the font `Calibri`).
```

File: content/css/concepts/typography/terms/font-style/font-style.md [hunk 11]
```diff
@@ -24,17 +25,51 @@ font-style: <font-style-value>;
...
-Set the `p` tag to `italic`:
+`italic`: text will be tilted, and some characters will be replaced with a specialized tilted character (e.g. "a" to "ɑ" in the font `Calibri`).
 
 ```css
 p {
   font-style: italic;
 }
 ```
+
+`oblique`: gives text a minor slant to the font without changing any characters. It is less supported and may not always be available.
```
Comment (avdhoottt) [inline content/css/concepts/typography/terms/font-style/font-style.md:51]:
```suggestion
`oblique`: It gives text a minor slant to the font without changing any characters. It is less supported and may not always be available.
```

File: content/css/concepts/typography/terms/font-style/font-style.md (hunk 9 above)
Comment (avdhoottt) [inline content/css/concepts/typography/terms/font-style/font-style.md:35]:
```suggestion
`normal`: The text will be in it's basic design. This is the default style in Cascading Style Sheets (CSS).
```

File: content/css/concepts/typography/terms/font-style/font-style.md [hunk 12]
```diff
@@ -24,17 +25,51 @@ font-style: <font-style-value>;
...
+
+`inherit`: text will take on the style used by the parent element.
+
+```css
+p {
+  font-style: inherit;
+}
+```
+
+`initial`: set the text `font-style` to its default style, similar to `normal`.
```
Comment (avdhoottt) [inline content/css/concepts/typography/terms/font-style/font-style.md:67]:
```suggestion
`inherit`: text will take on the style used by the parent element.

```css
p {
  font-style: inherit;
}
```

`initial`: Sets the text `font-style` to its default style, similar to `normal`.
```

Comment (avdhoottt) [inline content/css/concepts/typography/terms/font-style/font-style.md:67]:
```suggestion
`initial`: Sets the text `font-style` to its default style, similar to `normal`.
```

File: content/css/concepts/typography/terms/font-style/font-style.md [hunk 13]
```diff
@@ -24,17 +25,51 @@ font-style: <font-style-value>;
...
+
+`oblique`: It gives text a minor slant to the font without changing any characters. It is less supported and may not always be available.
+
+```css
+p {
+  font-style: oblique;
+}
+```
+
+`inherit`: text will take on the style used by the parent element.
```
Comment (avdhoottt) [inline content/css/concepts/typography/terms/font-style/font-style.md:59]:
```suggestion
`inherit`: The text will take on the style used by the parent element.
```

**Review by avdhoottt — APPROVED** [review]
LGTM!! Nice entry!!

**Pull request merged**